apps/marketing/src/app/page.tsx

```
@@ -1,27 +1,596 @@
+          </p>
+        </div>
+
+        <div className="mt-8 overflow-hidden">
+          <div className=" py-2 text-xs text-muted-foreground">TypeScript</div>
+          <div className="rounded-[18px] bg-primary/20 p-1">
+            <div className="rounded-[14px] bg-primary/20 p-0.5 shadow-sm">
+              <div className="bg-background rounded-xl overflow-hidden">
+                <CodeBlock
+                  lang="typescript"
+                  children={code}
+                  className="p-4 rounded-[10px]"
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)